jwst/fringe/fringe.py

```diff
@@ -53,7 +53,7 @@ def apply_fringe(input_model, fringe):
 
     Returns
     -------
-    output_model : jwst.datamodel.IFUImageModel
+    output_model : jwst.datamodels.IFUImageModel
```

Comment:
Looks like many of the new fixes for jwst.datamodel.* need backticks.
```suggestion
    output_model : `~jwst.datamodels.IFUImageModel`
```